src/ray/gcs/pubsub/gcs_pub_sub.h

```diff
@@ -132,5 +133,26 @@ class GcsSubscriber {
   const std::unique_ptr<pubsub::SubscriberInterface> subscriber_;
 };
 
+// This client is only supposed to be used from Cython / Python
+class RAY_EXPORT PythonGcsPublisher {
+ public:
+  explicit PythonGcsPublisher(const std::string &gcs_address);
+  Status Connect();
```

Comment:
Can you add docstring for the interface?

Reply:
fixed